.storybook/manager.ts

```diff
@@ -4,8 +4,14 @@ export const mode = process.env.STORYBOOK_MODE; // defined on CI script
 if (env === 'production' && mode === 'production')
   window.STORYBOOK_GA_ID = process.env.STORYBOOK_GA_ID;
 
+// everytime Storybook changes the document title, replace it with "Castor"
+new MutationObserver(() => {
+  if (document.title.endsWith('Storybook'))
+    document.title = document.title.slice(0, -9) + 'Castor';
+}).observe(document.querySelector('title'), { childList: true });
+
 declare global {
   interface Window {
-    STORYBOOK_GA_ID: string;
+    STORYBOOK_GA_ID?: string;
```

Comment:
And a little unrelated change 🙈

Reply:
Whoops, staging parts of the same file in different commits gets me every time.
Will revert this line